metadata.rb

```diff
@@ -5,7 +5,7 @@
 license 'Apache-2.0'
 description 'Allows for fetching and executing compliance profiles, and reporting its results'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '7.8.0'
+version '8.0.0'
```

Comment:
Should update the README support matrix entry for Chef Client 15 to `>= 8.0.0` as well

Reply:
Yes, pushed a commit for that, thank you!